src/main/java/com/hemant/jlambda/App.java

```diff
@@ -36,7 +38,12 @@
     @ArgGroup(exclusive = true, multiplicity = "1")
     private ParsedIntent intent;
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
+        if (args.length > 1 && args[1].equals(".")) {
+            String current = new java.io.File( "." ).getCanonicalPath();
+            System.out.println("Current dir:"+current);
+            args[1] = current;
+        }
```

Comment:
@ushachengan We should try to keep this class clean , all validations/processing should be abstracted away.
I would suggest you add a method in `EventsRunner.java` called `normalizePath`
It could have the below
```
private String normalizePath(String path) {
        if (path.equalsIgnoreCase(".")) {
            return System.getProperty("user.dir");
        }
        return path;
    }
```
And now you would want to call this method at all places where `Build()` `Publlish()` 
`Generate()` constructors are called like so
```new Build(normalizePath(ParsedIntent.build))```

Reply:
@hemantgs makes sense, Let me fix it and test it out. Thanks